src/blockdev.rs

```diff
@@ -225,6 +225,22 @@ pub fn try_discard_all(file: &mut File) -> Result<bool> {
 ioctl_none!(blkrrpart, 0x12, 95);
 ioctl_write_ptr_bad!(blkdiscard, request_code_none!(0x12, 119), [u64; 2]);
 
+pub fn udev_settle_until_partition_appears(device: &str, label: &str) -> Result<()> {
+    eprint!("Waiting for partition label {} to come online", label);
```

Comment:
If we want to report this to the user (and I'm not 100% on that) it should be in `write_disk()` (and an `eprintln!`) rather than in utility code.

Probably quotes around the `{}`, too.